BraveRewardsUI/Wallet/WalletViewController.swift

```diff
@@ -105,6 +88,11 @@ class WalletViewController: UIViewController, RewardsSummaryProtocol {
     if presentedViewController == nil {
       navigationController?.setNavigationBarHidden(false, animated: animated)
     }
+    stopNotificationObserver()
```

Comment:
Also is there a reason to not update the view anyways even if its not on screen? If a user pops back from settings it would just be there?

Reply:
Yeah, since we dont observe for deletedNotification, if any notificationView that may require an action is added and the user performs the action that deletes this notification from ledger will not hide the notificationView. The user has to manually close this notification